tests/acceptance/features/bootstrap/SpacesContext.php

```diff
@@ -3596,22 +3596,41 @@ public function searchResultShouldContainSpace(string $user, string $spaceName):
 	}
 
 	/**
-	 * @When /^user "([^"]*)" sends PROPFIND request to space "([^"]*)" using the WebDAV API$/
-	 * @When /^user "([^"]*)" sends PROPFIND request from the space "([^"]*)" to the resource "([^"]*)" using the WebDAV API$/
+	 * @When /^user "([^"]*)" sends PROPFIND request to space "([^"]*)" with depth "([^"]*)" using the WebDAV API$/
```

Comment:
we can use this step for default depth:
`When user "Brian" sends PROPFIND request from the space "Shares" using the WebDAV API`
```suggestion
         * @When /^user "([^"]*)" sends PROPFIND request to space "([^"]*)" using the WebDAV API$/
	 * @When /^user "([^"]*)" sends PROPFIND request to space "([^"]*)" with depth "([^"]*)" using the WebDAV API$/
```

Reply:
I see the step below

Reply:
Can they be combined? or could put these together
```php
@When /^user "([^"]*)" sends PROPFIND request to space "([^"]*)" using the WebDAV API$/
@When /^user "([^"]*)" sends PROPFIND request to space "([^"]*)" with depth "([^"]*)" using the WebDAV API$/
```